schema.sql

```diff
@@ -26,3 +26,35 @@ ALTER TABLE animals ADD COLUMN  species_id INT;
 ALTER TABLE animals ADD COLUMN owner_id INT;
 ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id);
 ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY(owners_id) REFERENCES owners(id);
+
+CREATE TABLE vets (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(30) NOT NULL,
+    age INT,
+    date_of_graduation DATE,
+    PRIMARY KEY(id));
+
+CREATE TABLE specializations (
+    species_id INT,
+    vets_id INT,
+    PRIMARY KEY(species_id, vets_id));
+
+CREATE TABLE visits (
+    animals_id INT,
+    vets_id INT,
+    date DATE,
+    PRIMARY KEY(animals_id, vets_id, date));
```

Comment:
Great job @Grandi0z 🤝

- [ ] As stated in the project requirements, kindly ensure to add the `visit_date` column to the `visits` table so as to track the date of the visit.